mx.sulong/mx_sulong.py

```diff
@@ -1155,6 +1175,8 @@ def checkNoHttp(args=None):
     'su-travis-sulong' : [travisTestSulong, ''],
     'su-travis-jruby' : [travisJRuby, ''],
     'su-travis-argon2' : [travisArgon2, ''],
+    'su-ecj' : [runEcjTestCases, ''],
+    'su-checkcode' : [checkCode, ''],
```

Comment:
Can we rename the command to `su-basic-checkcode` or so? Otherwise the command name suggests that it executes all our linter tools.